### DISCUSSION TITLE
chezmoi: /opt/homebrew/etc/homebrew/brew.env: not in destination directory (/Users/Elise)

### DISCUSSION THREAD

**Discussion by GETandSELECT** [twpayne/chezmoi#3786] [Q&A] [answered]
Hi

I read in other discussion that it's not a good idea to add root owned files like /etc

this file is not root owned

```
chezmoi add brew.env
chezmoi: /opt/homebrew/etc/homebrew/brew.env: not in destination directory (/Users/Elise)

ls -al  /opt/homebrew/etc/homebrew/brew.env
.rw-r--r-- Elise admin 194 B Wed May 29 18:52:53 2024  /opt/homebrew/etc/homebrew/brew.env
```

Thank you

**Comment by halostatue** [comment]
The problem is not root-owned, but outside of `$HOME`. You could have a *second* chezmoi repository for homebrew where `$HOME` gets simulated as `/opt/homebrew`.

I’m not sure what you’re trying to do, as my installation of Homebrew does not have a `brew.env` in `/opt/homebrew/etc/homebrew/brew.env`.

**Reply by GETandSELECT** [reply]
It's custom config for brew

```
cat /opt/homebrew/etc/homebrew/brew.env
HOMEBREW_FORBIDDEN_CASKS=openjdk
HOMEBREW_FORBIDDEN_FORMULAE=openjdk
HOMEBREW_CASK_OPTS="--no-quarantine"
HOMEBREW_CLEANUP_MAX_AGE_DAYS=2
HOMEBREW_NO_ANALYTICS=1
HOMEBREW_NO_UPDATE_REPORT_NEW=1
```

How-To add a  second chezmoi repository for /opt/homebrew? It's overkill?

**Reply by bradenhilton** [reply, marked as answer]
@GETandSELECT Why does it need to be system-wide? You can create a [user-specific `brew.env`](https://docs.brew.sh/Manpage#environment).

**Reply by halostatue** [reply]
Huh. I learned something and can simplify my chezmoi configuration to use this.
